src/lambda_function/s3_event_config/app.py

```diff
@@ -40,14 +48,111 @@ def lambda_handler(event, context):
         raise KeyError(err_msg)
 
 
+def get_existing_bucket_notification_configuration_and_type(
+    s3_client: boto3.client, bucket: str, destination_type: str
+) -> typing.Tuple[dict, list]:
```

Comment:
This would be more readable as a named tuple or dict.

Reply:
I added this to: https://github.com/Sage-Bionetworks/recover/pull/87/commits/b473c56fcd1a4c4d35131fcf366c740e0564fee7 - Is this what you had in mind?

Reply:
This works! I oftentimes use dict to return multiple results in a clear way but the way typing works with a class is pretty nice.